test/univariate/continuous/exponential.jl

```diff
@@ -0,0 +1,4 @@
+
+test_cgf(Exponential(1), Any[0.9, -1, -100f0, -1e6])
```

Comment:
Maybe it would simpler to use tuples here?
```suggestion
test_cgf(Exponential(1), (0.9, -1, -100f0, -1e6))
```